xbmc/interfaces/legacy/WindowXML.h

```diff
@@ -44,7 +44,7 @@ namespace XBMCAddon
     ///                             skins path to look in for the xml.
     ///                             (default='Default')
     /// @param defaultRes           [opt] string - default skins resolution.
-    ///                             (default='720p')
+    ///                             (1080i, 720p, ntsc16x9, ntsc, pal16x9 or pal. default='720p')
```

Comment:
Where is this defined in the code? Maybe we can just link to it if it is an enumeration and avoid changing it all the time

Reply:
https://github.com/xbmc/xbmc/blob/master/xbmc/addons/Skin.cpp#L354-L372
i wouldn't worry to much about it, this code hasn't changed in 10 years.

Reply:
Yeap, makes sense. Ok for me